src/bin/firebase.js

```diff
@@ -2,6 +2,8 @@
 "use strict";
 
 // Make check for Node 6, which is no longer supported by the CLI.
+import { tryStringify } from "../utils";
```

Comment:
move this to below the `updateNotifier` logic.

The top portion of this file is only the Node6 checks - everything else after is evaluated after that check passes.